weekly-mission(week2)/css/signin.css

```
@@ -1,158 +1,142 @@
 @import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard-dynamic-subset.css');
-
-* {
-    box-sizing: border-box;
-}
-
-:root {
-    --primary: #6D6AFE;
-    --red: #FF5B56;
-    --black: #111322;
-    --white: #ffffff;
-    --gray-5: #3E3E43;
-    --gray-4: #9FA6B2;
-    --gray-3: #CCD5E3;
-    --gray-2: #E7EFFB;
-    --Linkbrary-bg: #F0F6FF;
-    --gra-purpleblue-to-skyblue: linear-gradient(91deg, #6D6AFE 0.12%, #6AE3FE 101.84%);
-}
+@import url('color.css');
+@import url('reset.css');
```

여기서는 문제가 없기는 한데요~
보통 reset.css같은 경우에는 아래와 같은 흐름이기 때문에 혹시 나중에 실수를 방지하기 위해 가장 앞순서로 import를 해주시면 좋을 것 같아요!

1. 브라우저의 기본 스타일을 초기화할거야!
2. 그 이후에 나만의 스타일을 적용할거야!
```suggestion
@import url('reset.css');
@import url('color.css');
```